benchmarks/src/main/java/org/apache/druid/benchmark/FlattenJSONBenchmark.java

```diff
@@ -92,7 +92,8 @@ public void prepare() throws Exception
   public Map<String, Object> baseline(final Blackhole blackhole)
   {
     Map<String, Object> parsed = flatParser.parseToMap(flatInputs.get(flatCounter));
-    for (String s : parsed.keySet()) {
+    for (Map.Entry<String, Object> entry : parsed.entrySet()) {
```

Comment:
This can be just an iteration over `parsed.values()`. Similar in many other places in this PR.